recipes/ecell4_base/bld.bat

```diff
@@ -0,0 +1,3 @@
+"%PYTHON%" setup.py install
```

Comment:
Do the same here

Reply:
Can you remove this line?

Reply:
Sorry, I didn't notice that there is still the old line...